src/control/lib/spdk/ctests/SConscript

```diff
@@ -0,0 +1,32 @@
+#!/bin/env python
+"""Build go-spdk bindings C tests"""
+from __future__ import print_function
+import os
+from os.path import join, isdir
+import daos_build
+
+def scons():
+    """Execute build"""
+    Import('senv', 'prereqs')
+
+    unit_env = senv.Clone()
+    prereqs.require(unit_env, 'pmdk', 'spdk', 'hwloc', 'cmocka')
+
+    # equivalent to -I flags for spdk/lib/nvme and spdk/include for ut mocking
+    spdk_src = unit_env.subst("$SPDK_SRC")
+    unit_env.AppendUnique(CPPPATH=[join(spdk_src, 'lib', 'nvme'),
+                                   join(spdk_src, 'include')])
+
+    if isdir(spdk_src) and len(os.listdir(spdk_src)) != 0:
```

Comment:
Shouldn’t it check for the library we are linking?  Theoretically the source could be there but never built

Reply:
we are linking many of the libraries but this is checking that we have the source so we can link the  internals for use in mocking the unit tests. we could check for the libraries but that wouldn't verify that we have the internal headers available needed to build the test binary.

Reply:
addressed in follow-up: https://github.com/daos-stack/daos/pull/1505